.circleci/config.yml

```diff
@@ -254,7 +254,11 @@ jobs:
       - install-kubectl
       - install-sshfs
       - run:
-          command: make test
+          command: |
+            echo "127.0.0.1 beta-app.datawire.io" | sudo tee -a /etc/hosts
+            echo "127.0.0.1 app.getambassador.io" | sudo tee -a /etc/hosts
```

Comment:
Perhaps add `metriton.datawire.io` to the list to ensure that broken reporting doesn't mess things up.